src/main/java/net/tiagofar78/prisonescape/commands/JoinSubcommand.java

```diff
@@ -0,0 +1,44 @@
+package net.tiagofar78.prisonescape.commands;
+
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import net.tiagofar78.prisonescape.game.PrisonEscapeGame;
+import net.tiagofar78.prisonescape.managers.GameManager;
+import net.tiagofar78.prisonescape.managers.MessageLanguageManager;
+
+public class JoinSubcommand implements PrisonEscapeSubcommandExecutor {
+
+    @Override
+    public boolean onCommand(CommandSender sender, String label, String[] args) {
+        MessageLanguageManager messages = MessageLanguageManager.getInstance(sender.getName());
+
+        if (!(sender instanceof Player)) {
+            sender.sendMessage(messages.getOnlyPlayersCanUseThisCommandMessage());
+            return true;
+        }
+
```

Comment:
Missed check for permission. It is a good practice to associate permissions to commands even if commands are supposed to be able to be used by everyone. Use play permission defined in PrsionEscape class.